testing/src/main/java/io/stargate/it/http/MetricsTest.java

```diff
@@ -108,6 +108,54 @@ public void restApiHttpRequestMetrics() throws IOException {
             });
   }
 
+  @Test
+  public void docsApiHttpRequestMetrics() throws IOException {
+    // call the rest api path with target header
+    String path = String.format("%s:8082/v2/namespaces/some", host);
+    OkHttpClient client = new OkHttpClient().newBuilder().build();
+    Request request =
+        new Request.Builder()
+            .url(path)
+            .get()
+            .addHeader(TagMeHttpMetricsTagProvider.TAG_ME_HEADER, "test-value")
+            .build();
+
+    int status = execute(client, request);
+
+    await()
+        .atMost(Duration.ofSeconds(5))
+        .untilAsserted(
+            () -> {
+              String result =
+                  RestUtils.get("", String.format("%s:8084/metrics", host), HttpStatus.SC_OK);
+
+              List<String> lines =
+                  Arrays.stream(result.split(System.getProperty("line.separator")))
```

Comment:
(nitpick) Probably not an actual problem but relying on local line separate default is usually not a good idea; and since line separators are either `\n`, `\r` or `\r\n` might make sense to rather just split with regexp of `[\\r\\n]`.

But more to the point would probably make sense to encapsulate something like "findHttpHeaderLines(String response, String headerType)" at some point to make tests more compact.